packages/frontend/src/utils/project/isProjectUnderReview.ts

```diff
@@ -0,0 +1,20 @@
+import { Bridge, BridgeRiskView, Layer2, Layer2RiskView } from '@l2beat/config'
+
+export function isProjectUnderReview(project: Layer2 | Bridge): boolean {
```

Comment:
This function needs a comment that in case of adding a new section there is a need to also include it in this function